review/tests.py

```diff
@@ -46,3 +46,25 @@ def test_get_review_by_daycare_id(self, number_of_reviews, create_dog_owner_user
         assert len(reviews_by_id_list) == number_of_reviews
         for position, review in enumerate(reviews_by_id_list):
             assert review == review_list[position]
+
+    @pytest.mark.parametrize('daycare_id, expected_rating', [(1, 2), (2, 4.67)])
+    def test_get_average_rating_for_test_data(self, daycare_id, expected_rating):
+        average_rating = Review.get_average_rating_by_daycare_id(daycare_id)
+
+        assert float(expected_rating) == average_rating
+
+    def test_get_average_rating_for_invalid_daycare(self):
+        average_rating = Review.get_average_rating_by_daycare_id(-1)
+
+        assert 0 == round(average_rating, 2)
```

Comment:
Why use round here, in this case, shouldn't avrage_reveiw return 0 ?

Reply:
Fixed :)